oscar/slurm-array-job.md

```diff
@@ -0,0 +1,29 @@
+# Parallelize Computation with Slurm Array Jobs
+
+## R example
+Slurm array jobs can submit multiple identical jobs with different task ID's.
+This task ID can be passed as argument to an R script and used e.g. to run a script on a different file.
+The following code snippets show a basic setup to make use of Slurm Array Jobs in this context.
+
+### slurm-job.sh
+```bash
+#!/bin/bash
+#SBATCH --time=0-01:00 # Runtime in D-HH:MM
+#SBATCH --job-name="arrayJob"
+#SBATCH --account=ccmb-condo
+#SBATCH --mem-per-cpu=1G
+#SBATCH --cpus-per-task=32
+#SBATCH -n 1
+#SBATCH --array=1-8
+
+module load R/4.0.5
+
+Rscript --vanilla "path/to/script.R" "${SLURM_ARRAY_TASK_ID}"
+```
+### script.R
+```R
+args = commandArgs(trailingOnly=TRUE)
+task_id <- as.integer(args[1])
+data_files <- list.files(path = file.path("data/dir"), pattern = 'regex file pattern')
```

Comment:
Is this an R thing to have inconsistent usage of quotations? Or more likely do I not know when to use `'s` vs `"s` ?

Reply:
Good catch. In R they are interchangable most of the time.